# DP: Description: we need a longer timeout for slow machines
# DP: Author: Guido Guenther <agx@sigxcpu.org>
# DP: Upstream status: In CVS
# DP: Date: 2002-10-16

--- libc/stdio-common/tst-rndseek.c.orig	Wed Oct 16 17:02:06 2002
+++ libc/stdio-common/tst-rndseek.c	Wed Oct 16 17:02:16 2002
@@ -10,7 +10,7 @@
 
 static int do_test (void);
 #define TEST_FUNCTION do_test ()
-#define TIMEOUT 4
+#define TIMEOUT 10
 
 #include "../test-skeleton.c"
 
